packages/rainbowkit/src/wallets/walletConnectors/zerionWallet/zerionWallet.ts

```diff
@@ -0,0 +1,118 @@
+/* eslint-disable sort-keys-fix/sort-keys-fix */
+import { InjectedConnector } from 'wagmi/connectors/injected';
+import { Chain } from '../../../components/RainbowKitProvider/RainbowKitChainContext';
+import { isAndroid } from '../../../utils/isMobile';
+import { Wallet } from '../../Wallet';
+import { getWalletConnectConnector } from '../../getWalletConnectConnector';
+
+export interface ZerionWalletOptions {
+  chains: Chain[];
+  shimDisconnect?: boolean;
+}
+
+export const zerionWallet = ({
+  chains,
+  shimDisconnect,
+}: ZerionWalletOptions): Wallet => {
+  const isZerionInjected =
+    typeof window !== 'undefined' &&
+    typeof window.ethereum !== 'undefined' &&
+    // `isRainbow` needs to be added to the wagmi `Ethereum` object
+    // @ts-expect-error
+    window.ethereum.isZerion;
+
+  const shouldUseWalletConnect = !isZerionInjected;
+
+  return {
+    id: 'zerion',
+    name: 'Zerion',
+    iconUrl: async () => (await import('./zerionWallet.svg')).default,
+    iconAccent: '#2962ef',
+    iconBackground: '#fff',
+    installed: !shouldUseWalletConnect ? isZerionInjected : undefined,
+    downloadUrls: {
+      browserExtension: '#',
```

Comment:
Can we add a URL here, even if it's a redirect/placeholder? Example `https://zerion.io/download`. We will re-consider multiple browser store links in the near future, but we like that Phantom aggregates download options by linking to: `https://phantom.app/download`